docs/how-to/define-proto-and-service-in-a-shared-library.rst

```diff
@@ -39,4 +43,64 @@ In this specific case, my_external_library is an external library.
       app_registry.register(ExternalService)
 
 By defining here, "to_root_grpc=True", when you generate your protos, they will be added in a folder with the name and path defined in your 
-setting grpc_settings.ROOT_GRPC_FOLDER. (by default grpc_folder at the root of your application).
\ No newline at end of file
+setting grpc_settings.ROOT_GRPC_FOLDER. (by default grpc_folder at the root of your application).
+
+
+# :TODO: 
+Comment:
```

Comment:
remove this comment and put it in a other PR to discuss on it the time we figure out what the recommendations